bin/deploy-hdfs.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# requires ansible and ansible-runner: pip3 install --user ansible ansible-runner
+import os
+from argparse import ArgumentParser
+from configparser import ConfigParser
+from tempfile import NamedTemporaryFile
+
+import ansible_runner
```

Comment:
Can you check whether the import fails and then give an error containing the first line in this file?

Reply:
Added a try catch block to catch import failure